pkg/broker/broker.go

```diff
@@ -186,18 +190,53 @@ func (a AnsibleBroker) Bootstrap() (*BootstrapResponse, error) {
 		a.log.Error("Something went real bad trying to delete batch specs... - %v", err)
 		return nil, err
 	}
+	specs = []*apb.Spec{}
 
-	if specs, imageCount, err = a.registry.LoadSpecs(); err != nil {
-		return nil, err
+	//Load Specs for each registry
+	registryErrors := []error{}
+	for _, r := range a.registry {
+		s, i, err := r.LoadSpecs()
```

Comment:
I would've used `c` or `cnt` for count instead of `i`. I thought the i was an index but based on the imageCount it's a total.